BezierKit/BezierKitTests/PolynomialTests.swift

```diff
@@ -73,10 +73,20 @@ class PolynomialTests: XCTestCase {
         XCTAssertEqual(roots[1], 2, accuracy: accuracy)
     }
 
+//    func testDegree3RootExactlyZero() {
+//        // root is exactly t = 0 (at the start of unit interval),
+//        // so may be accidentally discarded due to numerical precision
+//        let polynomial = BernsteinPolynomial3(b0: 0, b1: 96, b2: -24, b3: -36)
+//        let roots = findRoots(of: polynomial, between: 0, and: 1)
+//        XCTAssertEqual(roots.count, 2)
+//        XCTAssertEqual(roots[0], 0.0)
+//        XCTAssertEqual(roots[1], 2.0 / 3.0, accuracy: accuracy)
+//    }
+
```

Comment:
Can we get it so this unit test works?

Reply:
I don't think so. It's a problem in droots where a rounding error is introduced in the cubic solver. The Bezier clipping algorithm probably wouldn't have this issue. But I don't think it causes problems in the app due to judicious usages of epsilon and fallbacks when we miss this root.